### DISCUSSION TITLE
Writing text vertically

### DISCUSSION THREAD

**Discussion by Cyberduke01** [cduck/drawSvg#52] [Q&A] [answered]
Hi, I am trying to write text vertically. 

Using rotate=90 as a parameter in the Text(as per the example below), turns every letter 90 degrees, but not the whole text.  

`d.append(draw.Text(text,textSize,label[0],label[1],rotate=90, fill='red'))  `

**Comment by cduck** [comment, marked as answer]
Use the `transform` parameter with `'rotate(...)'`: https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform
